docs/strategies/fallback.md

```diff
@@ -12,48 +12,48 @@
 
 <!-- snippet: fallback -->
 ```cs
-// Add a fallback/substitute value if an operation fails.
-new ResiliencePipelineBuilder<UserAvatar>()
-    .AddFallback(new FallbackStrategyOptions<UserAvatar>
-    {
-        ShouldHandle = new PredicateBuilder<UserAvatar>()
-            .Handle<SomeExceptionType>()
-            .HandleResult(r => r is null),
-        FallbackAction = args => Outcome.FromResultAsValueTask(UserAvatar.Blank)
-    });
+// A fallback/substitute value if an operation fails.
+var optionsSubstitute = new FallbackStrategyOptions<UserAvatar>
```

Comment:
```suggestion
fallbackOptions = new FallbackStrategyOptions<UserAvatar>
```

And just keep re-assgning?

Reply:
I suggested against this in a previous comment as if you copy the whole snippet as a user it's just mutating and not functionally doing anything and might lead to people writing code like this and then asking questions when it doesn't work.